spikefeatures/features.py

```diff
@@ -259,15 +281,19 @@ def _get_slope(x, y):
 
 def _get_trough_and_peak_idx(waveform):
     """
-    Return the indices into the input waveforms of the detected troughs (minimum of waveform)
-    and peaks (maximum of waveform, after trough).
+    Return the indices into the input waveforms of the detected troughs
+    (minimum of waveform) and peaks (maximum of waveform, after trough).
 
     Assumes negative troughs and positive peaks
+
+    Returns 0 if not detected
     """
     trough_idx = np.argmin(waveform, axis=1)
-    peak_idx = np.empty(trough_idx.shape, dtype=int)
+    peak_idx = np.empty(trough_idx.shape, dtype=int)  # int, these are used for indexing
```

Comment:
can you initialize to `np.nan` here as well? Empty is dangerous because it gets random values